cg/server/admin.py

```diff
@@ -7,7 +7,8 @@
 
 
 class BaseView(ModelView):
-
+    can_export = True
```

Comment:
I see that we have added this in the BaseView, reason?

Reply:
I got "Tablib dependency not installed. " in all views I tried, some dependency missing?

Reply:
ok, gone